enhancements/windows-containers/container-runtime-containerd.md

```diff
@@ -0,0 +1,213 @@
+User can choose to uninstall Docker or use it for non-kubernetes related requirements. As docker installation is
+handled by the user, it is up to the user to decide if Docker should continue to run or not.
+When it comes to MachineSet, we can choose a VM image which doesn't have Docker. Containerd will be the only runtime.
+Even if Docker gets installed at any time, it can co-exist with containerd. 
+steps to install containerd as service
+    *scp containerd/related executables into Windows VM
+    *copy the files in appropriate folder
+    *create containerd config file
+    *run below commands in PowerShell cmdlet
+    *run extra command for performance `Add-MpPreference -ExclusionProcess "$Env:containerd.exe"`
+    *register containerd as service
+    *start containerd as a Windows service
+
+## Network changes
+Current CNI/IPAM will be used for containerd and no change in HNS-Network and HNS-Endpoint creation
+steps. The config file which will be used by containerd will point to same CNI/IPAM executables.
+
+## WMCO feature flag
+Containerd feature flag will be used to enable this feature. This will be enabled by default in
+community operator for customers and developers to try it out.  This will become the default for OpenShift
+4.11. At any given point in time, we have one runtime support and do not allow switching between runtime.
+
+## logging
+Containerd can be started with parameters in which we can enable logging and specify the file path to
+log warnings/errors. Log files will be stored at c:\var\log\containerd
+
+## upgrade
+When a cluster is upgraded OLM will switch to using a new Red Hat operators
+index. Because WMCO is named the same in both indexes, OLM will upgrade WMCO
+from the previous version, up to the latest version available in the new
+cluster. For further details please refer [WMCO-Upgrade](https://github.com/openshift/enhancements/blob/master/enhancements/windows-containers/windows-machine-config-operator-upgrades.md)
+
+The procedure for an upgrade is as follows:
+1) MachineSet
+   * As part of upgrade Machine object gets deleted, resulting in the drain and deletion of the Windows node.
+   * New VM will be created.
+   * The upgraded WMCO instance will then be able to configure VM that will be created with containerd as 
+   default runtime.
+   * Once the Windows node joins kubernetes cluster, user can deploy pods on the Windows node.
+2) BYOH ( Bring Your Own Host)
+   * upgrade process will uninstall all components that were installed by WMCO. 
+   * As part of the de-configuring process Windows OS-specific configurations like HNS-Network that were created by WMCO 
+     would be deleted. 
+   * Once cleanup is done, reinstall workflow will install containerd along with kubelet, kube-proxy, CNI and 
+     hybrid-overlay.
+   * Kuebelet will start using containerd as default runtime.
+   * Once the Windows node joins kubernetes cluster, user can deploy pods on the Windows node.
```

Comment:
This is repeated information. I suggest consolidating in one spot and linking where needed.

Reply:
removed from the proposal section and consolidated here.